### ISSUE TITLE
Use our forums for suggestions; not our github

### ISSUE THREAD

**Issue by Super3222** [yogstation13/yogstation#2116]
This is my attempt to bring the community together on game development, features, and how to handle moving forward.

https://forums.yogstation.net/index.php?threads/bring-back-the-suggestion-subforum.13093/

I will try to work with the Council on restoring our suggestion forum once more.

As of now, any more suggestions will be closed. The Council will need to re-open the suggestion forum to the public (and other tweaks I've suggested) for more suggestions to be publicized.

To be fair, it's been nice reading all of the suggestions you guys have come up with. I hope we can continue that development on the forums.

**Comment by Super3222** [comment]
https://github.com/yogstation13/yogstation/issues/2071

**Comment by 00aquir** [comment]
so... what is the advantage?

**Comment by Super3222** [comment]
Bigger audience. Suggestions will get more replies/discussion. Coders will be able to get real-time feedback on PR's through topics that players / contributors open up. Voting polls for removing / adding features.

**Comment by Dagdammit** [comment]
Seems like a damn solid move.

**Comment by Frostben** [comment]
Neato this is gonna be fun

**Comment by Kmc2000** [comment]
Features we'll lose:

Git is infinitely cleaner.

You can reference each request as you PR it.

Issues can be sorted with tags by maintainers.


If we go back to the forums, itll become a clusterfuck mess and only reach a smaller audience because only regulars tend to use the forums.


Against personally. We could have both but that'd be hell for the maintainers.

Perhaps an issue manager? in this case, who takes ideas from the forums and nags the coders?

Finally, so called bigger audience. 

What youre saying is more people like you know who making shitty meme suggestions like a mouse antag, but hey at least the get the chance to voice their opinions! D E M O C R A C Y.


Tldr, i dont see many advantages, and we will just devolve into a steam esque shitstorm of ideas fighting for attention because name more than one coder who'd check the forums and sift through the mentions.

/:popcorn:

**Comment by Frostben** [comment]
Oh shit that mouse antag actually sounds like a plausible idea, I'm going to brainstorm that for a few hours then wait for the new forum thread to post it there :P

**Comment by 00aquir** [comment]
we have a mouse ghost role suggestion, antag would be an overkill

**Comment by Frostben** [comment]
Which issue is that?

**Comment by 00aquir** [comment]
https://github.com/yogstation13/yogstation/issues/1998

**Comment by Frostben** [comment]
Thanks

**Comment by Drakkon** [comment]
reeeeee

**Comment by Super3222** [comment]
@Kmc2000 Unless git pulls off some amazing feature that can best what the forum does (polls, attracting new people, etc.) then it's not a good option to stay on it.

And think about - if the forums aren't getting any new people then the github definitely isn't.

PR's won't be affected by this. You can still talk about EVERYTHING a PR has to offer and even suggest things on the PR itself. This is only about issues.

Now back onto issues, git named them specifically _issues_ for a reason. They're not supposed to be elaborate discussions or feature requests. They are supposed to be there to talk about bugs.

We also had a suggestion forum opened up in the past. None of the things you're talking about happened when it was open. Maybe there were a few meme ideas, but decent ones are still the majority.

The only good thing we'll be missing is referencing PR's, and that can be easily compensated for... referencing the topics from the forums. Almost every other (ss13) repo does that same thing.
